tests/integration/popular_collections/generate.py

```diff
@@ -0,0 +1,41 @@
+"""Generate lists of most popular collections for each of the hardcoded DAACs."""
+
+from pathlib import Path
+
+import requests
+
+THIS_DIR = Path(__file__).parent
+
+
+def top_collections(*, provider: str, num: int = 100) -> list[str]:
+    response = requests.post(
+        "https://cmr.earthdata.nasa.gov/search/collections.json",
+        data={
+            "provider": provider,
+            "has_granules_or_cwic": True,
+            "include_facets": "v2",
+            "include_granule_counts": True,
+            "include_has_granules": True,
+            "include_tags": "edsc.*,opensearch.granule.osdd",
+            "page_num": 1,
+            "page_size": 100,
+            "sort_key[]": "-usage_score",
+        },
```

Comment:
I just copied this out of my network tab after doing a similar query in earthdata search client. I'm sure we can run an equivalent query with earthaccess.